components/dropdown/dropdown-content-mixin.js

```diff
@@ -801,6 +812,9 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 					</div>
 				</div>
 			</div>
+			<style>
+				${dropdownContentStyles(this.mobileBreakpointOverride)}
```

Comment:
Adding the styles using the <style> attribute on each render. We need to load it for each render rather than using the static styles() function (shared among all instances).

Reply:
The downside to this is that browser who supported constructible stylesheets won't be able to reuse these styles across all dropdowns.

An alternate but similar approach that might work:

Instead of using a media query in `dropdown-content-styles` (i.e. `@media (min-width: 616px) {`), could you use the presence of a CSS class or `data-` attribute on the host to trigger the mobile styles to kick in?

For example:

```css
:host([data-mobile][opened]) {
    animation: var(--d2l-dropdown-animation-name) 300ms ease;
    display: inline-block;
}
```

And then you could use the `mediaQueryList` to detect when things are a match, but you'd need to add an event listener on it like we did in fullscreen dialogs to be notified when it changes.

Reply:
Thanks Dave. I'll try that!

Reply:
Thanks Dave.  Yeah this is the route that I was thinking as well.